PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/CosmosServerlessSparkSubmitModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.azure.hdinsight.spark.run.configuration
+
+import com.intellij.openapi.project.Project
+import com.microsoft.azure.hdinsight.spark.common.ServerlessSparkSubmitModel
+import com.microsoft.azure.hdinsight.spark.common.SparkSubmissionParameter
+import com.microsoft.azuretools.utils.Pair
+import java.util.stream.Stream
+
+class CosmosServerlessSparkSubmitModel(project: Project) : ServerlessSparkSubmitModel(project) {

```

Comment:
Maybe `ServerlessSparkSubmitModel` should be renamed to `CosmosSparkSubmitModel` in anohter PR #2347